lom/Makefile

```diff
@@ -6,17 +6,24 @@ USER = $(shell id -un)
 UID = $(shell id -u)
 GUID = $(shell id -g)
 
-
-GO := /usr/local/go1.20.3/go/bin/go
-
+GOBIN_PATH := /usr/local/go1.20.3/go/bin
+GO := $(GOBIN_PATH)/go
 GOFMT := $(GO)fmt
 INSTALL := /usr/bin/install
 
+ifeq ($(GOPATH),)
+export GOPATH=/tmp/go
+endif
+
+VENDOR_DIR := vendor
 BUILD_DIR := build
 LIB_DIR := $(BUILD_DIR)/lib
 BIN_DIR := $(BUILD_DIR)/bin
 TEST_DIR := $(BUILD_DIR)/test
 INTEGRATION_TEST_DIR := integration_test
+GNMI_DIR := src/gnmi
```

Comment:
gnmi server need libzmq  library too right ?

Reply:
Every one who uses lib/libtelemetry -- needs it